docs/ovs_communication.adoc

```diff
@@ -120,14 +135,16 @@ b. start vswitchd: ovs-vswitchd -v --pidfile --detach \
 
 c. use cmd to init: ovs-vsctl --no-wait init
 ------------------------------------------------------------
-
+option 2 is start the openvswitch service directly which would do the same to option 1, btw option 2 is the neutron way
 TBD - XiaoDong, which way are we going use?
 
 . br-int, br-tun are created during init time, and recreated later in RPC loop if needed
-.. TBD - XiaoDong, are we going to follow the same?
+.. TBD - XiaoDong, are we going to follow the same? 
+... I agree
 
 . what happen when there are multiple physical NICs on the system, which NIC do we pick to hook up to br-tun, br-vlan, br-ex (if needed)?
 .. what is the exact command to connect the new port to the new br-tun, and enable encap/decap?
+... A: if you are talking about container as the end point, then they would use ovs-docker cmd which would eventually called ip link/add etc and ovs-vsctl add-port set interface type=vxlan is the one which would do encap/decap
```

Comment:
ok, got it.

Reply:
for br-tun, ovs would use the nic ending with '_l' paring with the same uuid inside the container veth,
for br-vlan, ovs would use vxlan_system_4789 (default)